### PR TITLE
Feature: add extraction of opacities tokens

### PR DESCRIPTION
**Pull request by flo-sch**
- refactor(tokens): implement switch case in processTokens
- feat(tokens): enable opacities tokens
    - allow "opacity" and "opacities" as a frame name
    - allow "float" and "percent" as a unit (from configuration)
    - add processing function to fetch and convert tokens
    - adjust default configuration and allow file + CLI overriding
    - include test files for all updates

@mikaelvesavuori I tested against a Figma document that I have write access to :)
Let me know if you want me to add the related frame in the generic template?

### CODE REVIEW

File: bin/functions/process/processTokens.mjs [hunk 1]
```diff
@@ -31,33 +32,80 @@ export function processTokens(sheet, name, config) {
...
+    case 'spacing':
+    case 'spacings': {
+      if (!config) throw new Error(errorProcessTokensNoConfig);
+      processedTokens = setupSpacingTokens(sheet, config.spacingUnit, config.remSize);
+      break;
+    }
+    case 'zindex':
+    case 'zindices': {
+      processedTokens = setupZindexTokens(sheet);
+      break;
```
Comment (mikaelvesavuori) [inline bin/functions/process/processTokens.mjs:107]: Thanks for the uncluttering work here!

File: __tests__/parseCliArgs.test.mjs [hunk 2]
```diff
@@ -214,6 +214,35 @@ test('It should return default value for fontUnit if passing invalid value (long
...
+  const unit = 'percent';
+  expect(parseCliArgs(['--opacitiesUnit', unit])).toEqual(
+    expect.objectContaining({
+      opacitiesUnit: unit
+    })
+  );
+});
+
+test('It should return the unit for opacitiesUnit if passing valid value (short-hand)', () => {
+  const unit = 'float';
```
Comment (mikaelvesavuori) [inline __tests__/parseCliArgs.test.mjs:230]: Would be good to have the new CLI args documented in README

File: testdata/tokens/opacities.mjs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// THIS FILE IS AUTO-GENERATED BY FIGMAGIC. DO NOT MAKE EDITS IN THIS FILE! CHANGES WILL GET OVER-WRITTEN BY ANY FURTHER PROCESSING.
+
+const opacities = {
+  opaque: 1,
+  disabled: 0.65,
+  semiOpaque: 0.5,
+  transparent: 0
+};
+
+export default opacities;
```
Comment (mikaelvesavuori) [inline testdata/tokens/opacities.mjs:10]: I would happily add your opacity boilerplate to the base document! Give me a URL :)

File: __tests__/parseCliArgs.test.mjs (hunk 2 above)
Comment (flo-sch) [reply __tests__/parseCliArgs.test.mjs:230]: Done, you can review :)

**Comment by mikaelvesavuori** [comment]
Merging

**Pull request merged**